Description: Fill libmtp.pc's "Requires" field with the proper information
 regarding the USB library against which it's compiled against.
Author: Alessio Treglia <alessio@debian.org>
Bug-Debian: http://bugs.debian.org/666713
Forwarded: https://sourceforge.net/tracker/?func=detail&aid=3513878&group_id=158745&atid=809061
---
 configure.ac |    3 +++
 libmtp.pc.in |    2 +-
 2 files changed, 4 insertions(+), 1 deletion(-)

--- libmtp.orig/configure.ac
+++ libmtp/configure.ac
@@ -114,6 +114,7 @@ AM_CONDITIONAL(MS_LIB_EXE, test x$ms_lib
 PKG_CHECK_MODULES(LIBUSB, libusb-1.0 >= 1.0.0, found_libusb1_pkg=yes, found_libusb1_pkg=no)
 if test x"$found_libusb1_pkg" = "xyes" ; then
    LIBUSB_VER=`pkg-config --modversion libusb-1.0`
+   LIBUSB_REQUIRES='libusb-1.0'
    AC_DEFINE(HAVE_LIBUSB1, [], [Have libusb 1.0])
 fi
 
@@ -132,6 +133,7 @@ if test x"$found_libusb1_pkg" = "xno" ;
       PKG_CHECK_MODULES(LIBUSB, libusb >= 0.1.8, found_libusb0_pkg=yes, found_libusb0_pkg=no)
       if test x"$found_libusb0_pkg" = "xyes" ; then
          LIBUSB_VER=`pkg-config --modversion libusb`
+         LIBUSB_REQUIRES='libusb'
          AC_DEFINE(HAVE_LIBUSB0, [], [Have libusb 0.x])
       else
          # Some systems don't use pkg-config for libusb, so double check.
@@ -151,6 +153,7 @@ fi
 
 AC_SUBST(LIBUSB_CFLAGS)
 AC_SUBST(LIBUSB_LIBS)
+AC_SUBST(LIBUSB_REQUIRES)
 AM_CONDITIONAL(LIBUSB1_COMPILE, test x"$found_libusb1_pkg" = "xyes")
 AM_CONDITIONAL(LIBUSB0_COMPILE, test x"$found_libusb0_pkg" = "xyes")
 AM_CONDITIONAL(LIBOPENUSB_COMPILE, test x"$ac_cv_lib_openusb_openusb_init" = "xyes")
--- libmtp.orig/libmtp.pc.in
+++ libmtp/libmtp.pc.in
@@ -8,7 +8,7 @@ includedir=@includedir@
 Name: libmtp
 Description: libmtp is a library for accessing Media Transfer Protocol devices
 Version: @VERSION@
-Requires: libusb
+Requires: @LIBUSB_REQUIRES@
 Conflicts:
 Libs: -L${libdir} -lmtp
 Libs.private: @LIBS@
